Makefile.PL

```diff
@@ -65,6 +65,7 @@ WriteMakefile(NAME => 'LaTeXML',
     'Image::Size'       => 0,
     'IO::String'        => 0,
     'IO::Handle'        => 0,
+    'IO::Socket'        => 0,
```

Comment:
Not strictly necessary to add `IO::Socket` today, but might as well future-proof our dependency list, given that it was Socket.c that was tripping up CI.